README.md

```diff
@@ -71,4 +71,4 @@ Feel free to contact us at our [Gitter channel](https://gitter.im/NITSkmOS/algo)
 ## Notes
 
 Only project maintainers should merge a PR.
-Other members can add their reviews to a PR but the merging should be done by only a project maintainer.
+Other members can add their reviews to a PR but the merging should be done by only a project maintainer.
```

Comment:
Now, what happened with this one?

Reply:
This must have happened since this branch had conflicts due to prior merging of other PRs.